exercises.py

```diff
@@ -1,28 +1,40 @@
+from typing import Dict
+
 """
 Recursion Problem Set: Coding Exercises
 """
 
 def reverse(s: str) -> str:
-    """
-    Reverse string using recursive approach
-    """
+	if len(s) == 1:
+		return s
+	else:
+		return reverse(s[1:]) + reverse(s[0])
 
 def fib_rec(n: int) -> int:
-    """
-    Recursive fibonacci
-    """
+	if n < 2:
+		return n
```

Comment:
Beautiful. Was going to question returning `n` for `n <2` but if we start our fibonacci sequence at 0 (meaning `0, 1, 1, 2, 3, 5, ...`) then this perfect.

We could also start our sequence at 1 and return `1` for `n<2`.